react/react_intro/README.md

```diff
@@ -232,55 +250,56 @@ function App() {
 export default App;
 ```
 
-Every time that you save your application, the browser window will reload the changes that you've made.  You should now see the following screen:
+Every time that you save your application, as long as your server is running, the browser window will reload to reflect the changes that you've made. You should now see the following screen:
```

Comment:
`as long as your server is running ` -- will this be clear to them? We might want to specify that this is the process that was started when they ran `npm start`

Reply:
I'm all for it. Just to add, most of the fellows in my class have been using live server for their html. So it probably won't feel like a weird change to them.